contrib/automation_tests/core/e2e_utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from pywinauto.base_wrapper import BaseWrapper
+
+
+class OrbitE2EError(RuntimeError):
+    pass
+
+
+def first(parent: BaseWrapper, control_type, name=None, name_contains=None,
```

Comment:
I was really struggling with this name :-) Generally, I absolutely agree. Then I had to type this function a few times (sometimes nested to find controls in a hierarchy)... since it could be used a lot, decided that a shorter name would make the code more readable and less annoying to type. But I'm on the fence about this.

Just wanted to give some rationale - if you both still think the longer name is better, I'll gladly change it.

Reply:
I think the longer name is preferable; if you disagree maybe consider 'find' which I think is more readable than 'first'.

Reply:
As a middle ground, it's now named `find_control`. As we discussed, I've also moved this to orbit_e2e and got rid of the e2e_utils.py.

Reply:
lgtm